tests/org.eclipse.n4js.generator.headless.tests/src/org/eclipse/n4js/generator/headless/tests/ScenarioTest.java

```diff
@@ -109,15 +137,20 @@ static String readFile(Path path)
 	 */
 	@Before
 	public void setUp() throws Exception {
-		//
+		subgeneratorsRegistry.register(ecmaScriptSubGenerator.get(), N4JSGlobals.N4JS_FILE_EXTENSION);
+		subgeneratorsRegistry.register(ecmaScriptSubGenerator.get(), N4JSGlobals.N4JSX_FILE_EXTENSION);
+		subgeneratorsRegistry.register(ecmaScriptSubGenerator.get(), N4JSGlobals.JS_FILE_EXTENSION);
```

Comment:
What about ` N4JSGlobals.JSX_FILE_EXTENSION`

Reply:
Done. N4jSX is removed